container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/engine_functions/create_engine.go

```diff
@@ -18,6 +18,7 @@ import (
 )
 
 const (
+	frontendPortSpec                            = 9711
```

Comment:
This should be an argument of the CreateEngine method like the `grpcPortNum` in order to centralize this number in a place as we did for the grpcPortNum

Reply:
will leave this for now; will create a ticket. good point. Will do it next PR - while adding this for k8s